.github/CODEOWNERS

```diff
@@ -352,6 +352,7 @@
 /test/kitchen/test/integration/win-user/ @DataDog/windows-agent
 /test/system/                           @DataDog/agent-shared-components
 /test/system/dogstatsd/                 @DataDog/agent-metrics-logs
+/test/benchmarks/apm_scripts/           @DataDog/agent-apm
```

Comment:
One last request - could you also add your team as owner of the `.gitlab/benchmars/benchmars.yml` file? I forgot to ask for it during the first review round. I'll approve when it's done. Thanks!

Reply:
Done! 👍